<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>HBorderSet</class>
 <widget class="QWidget" name="HBorderSet">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>522</width>
    <height>378</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QGroupBox" name="groupBox_5">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>10</y>
     <width>321</width>
     <height>315</height>
    </rect>
   </property>
   <property name="title">
    <string>边框</string>
   </property>
   <widget class="QPushButton" name="borderNoBtn">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>30</y>
      <width>32</width>
      <height>32</height>
     </rect>
    </property>
    <property name="focusPolicy">
     <enum>Qt::TabFocus</enum>
    </property>
    <property name="styleSheet">
     <string notr="true">outline: none;</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="icon">
     <iconset resource="saribbonresouce.qrc">
      <normaloff>:/icon/icon/BorderNone.png</normaloff>:/icon/icon/BorderNone.png</iconset>
    </property>
   </widget>
   <widget class="QPushButton" name="borderOutSideBtn">
    <property name="geometry">
     <rect>
      <x>170</x>
      <y>30</y>
      <width>32</width>
      <height>32</height>
     </rect>
    </property>
    <property name="focusPolicy">
     <enum>Qt::TabFocus</enum>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="icon">
     <iconset resource="saribbonresouce.qrc">
      <normaloff>:/icon/icon/BorderOutside.png</normaloff>:/icon/icon/BorderOutside.png</iconset>
    </property>
   </widget>
   <widget class="QPushButton" name="borderAllBtn">
    <property name="geometry">
     <rect>
      <x>230</x>
      <y>30</y>
      <width>32</width>
      <height>32</height>
     </rect>
    </property>
    <property name="focusPolicy">
     <enum>Qt::TabFocus</enum>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="icon">
     <iconset resource="saribbonresouce.qrc">
      <normaloff>:/icon/icon/BordersAll.png</normaloff>:/icon/icon/BordersAll.png</iconset>
    </property>
   </widget>
   <widget class="QPushButton" name="borderRightBtn">
    <property name="geometry">
     <rect>
      <x>230</x>
      <y>250</y>
      <width>32</width>
      <height>32</height>
     </rect>
    </property>
    <property name="focusPolicy">
     <enum>Qt::TabFocus</enum>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="icon">
     <iconset resource="saribbonresouce.qrc">
      <normaloff>:/icon/icon/BorderRight.png</normaloff>:/icon/icon/BorderRight.png</iconset>
    </property>
    <property name="checkable">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="borderLeftBtn">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>250</y>
      <width>32</width>
      <height>32</height>
     </rect>
    </property>
    <property name="focusPolicy">
     <enum>Qt::TabFocus</enum>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="icon">
     <iconset resource="saribbonresouce.qrc">
      <normaloff>:/icon/icon/BorderLeft.png</normaloff>:/icon/icon/BorderLeft.png</iconset>
    </property>
    <property name="checkable">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="borderVerBtn">
    <property name="geometry">
     <rect>
      <x>170</x>
      <y>250</y>
      <width>32</width>
      <height>32</height>
     </rect>
    </property>
    <property name="focusPolicy">
     <enum>Qt::TabFocus</enum>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="icon">
     <iconset resource="saribbonresouce.qrc">
      <normaloff>:/icon/icon/BorderInsideVertical.png</normaloff>:/icon/icon/BorderInsideVertical.png</iconset>
    </property>
    <property name="checkable">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="borderTopBtn">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>70</y>
      <width>32</width>
      <height>32</height>
     </rect>
    </property>
    <property name="focusPolicy">
     <enum>Qt::TabFocus</enum>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="icon">
     <iconset resource="saribbonresouce.qrc">
      <normaloff>:/icon/icon/BorderTop.png</normaloff>:/icon/icon/BorderTop.png</iconset>
    </property>
    <property name="checkable">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="borderHorBtn">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>140</y>
      <width>32</width>
      <height>32</height>
     </rect>
    </property>
    <property name="focusPolicy">
     <enum>Qt::TabFocus</enum>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="icon">
     <iconset resource="saribbonresouce.qrc">
      <normaloff>:/icon/icon/BorderInsideHorizontal.png</normaloff>:/icon/icon/BorderInsideHorizontal.png</iconset>
    </property>
    <property name="checkable">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="borderBottomBtn">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>210</y>
      <width>32</width>
      <height>32</height>
     </rect>
    </property>
    <property name="focusPolicy">
     <enum>Qt::TabFocus</enum>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="icon">
     <iconset resource="saribbonresouce.qrc">
      <normaloff>:/icon/icon/BorderBottom.png</normaloff>:/icon/icon/BorderBottom.png</iconset>
    </property>
    <property name="checkable">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QFrame" name="viewFrame">
    <property name="geometry">
     <rect>
      <x>93</x>
      <y>78</y>
      <width>181</width>
      <height>151</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
   </widget>
  </widget>
  <widget class="QGroupBox" name="groupBox_4">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>161</width>
     <height>261</height>
    </rect>
   </property>
   <property name="title">
    <string>直线</string>
   </property>
   <widget class="QTableWidget" name="lineSytleWidget">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>17</y>
      <width>140</width>
      <height>231</height>
     </rect>
    </property>
   </widget>
  </widget>
  <widget class="QComboBox" name="lineColorComboBox">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>300</y>
     <width>130</width>
     <height>25</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_9">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>280</y>
     <width>72</width>
     <height>15</height>
    </rect>
   </property>
   <property name="text">
    <string>颜色(C)</string>
   </property>
  </widget>
  <widget class="QPushButton" name="moreColorBtn">
   <property name="geometry">
    <rect>
     <x>145</x>
     <y>300</y>
     <width>35</width>
     <height>25</height>
    </rect>
   </property>
   <property name="text">
    <string>更多</string>
   </property>
   <property name="autoDefault">
    <bool>true</bool>
   </property>
   <property name="default">
    <bool>true</bool>
   </property>
  </widget>
 </widget>
 <resources>
  <include location="saribbonresouce.qrc"/>
 </resources>
 <connections/>
</ui>
